net-libs/tox/tox-0.2.7.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+#cmake-utils brings in cmake dependency
```

Comment:
To be honest, this is documented in the eclass. I don't think we should selectively comment on things that are documented elsewhere or the ebuild will quickly grow with unnecessary comments.